addons/mail/models/res_partner.py

```diff
@@ -234,6 +234,7 @@ def mail_partner_format(self, fields=None):
                 main_user = internal_users[0] if len(internal_users) > 0 else partner.user_ids[0] if len(partner.user_ids) > 0 else self.env['res.users']
                 data['user'] = {
                     "id": main_user.id,
+                    "notificationType": main_user.notification_type,
```

Comment:
Nitpicking: it'd be to put that property immediately on `data` rather than `data['user']`.
It's not great to deal nested objects, and field `user` on persona will likely be removed in the future. Inner-properties will be converted to immediate field on persona model.